style/components/nav.css

```diff
@@ -0,0 +1,79 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(80, 77, 77);
```

Comment:
maybe change the background color, it will look nicer and active

Reply:
sure I will look into it.